src/System.Text.Json/tests/Serialization/Object.ReadTests.cs

```diff
@@ -16,14 +16,47 @@ public static void ReadSimpleClass()
             obj.Verify();
         }
 
+        [Fact]
+        public static void ReadSimpleClassWithObject()
+        {
+            var obj = JsonSerializer.Parse<SimpleTestClassWithSimpleObject>(SimpleTestClassWithSimpleObject.s_json);
+            obj.Verify();
+            string reserialized = JsonSerializer.ToString(obj);
+
+            // Properties in the exported json will be in the order that they were reflected, doing a quick check to see that
+            // we end up with the same length (i.e. same amount of data) to start.
+            Assert.Equal(SimpleTestClassWithSimpleObject.s_json.Replace(" ", string.Empty).Length, reserialized.Length);
```

Comment:
If we start adding other whitespace like linefeeds or strings with spaces, then this Assert would fail. Is there a better way? I haven't been checking length - just re-verifying round-tripped data.

Reply:
We can make a more complicated whitespace stripper. Seems like a good check that we end up with the same amount of meaningful characters...